src/assets/news/1.md

```diff
@@ -0,0 +1,33 @@
+# Promoting Lemmy
+
+*Written by @nutomic, 2021-08-09*
```

Comment:
I've seen some markdown based blogging software (dont remember which) do it like this. Much easier to parse. And to be honest I dont think the author field is necessary.

```
title: Promoting Lemmy
date: 2021-08-09
author: @nutomic 
---

Content of post here (markdown format)
```

Reply:
Wayy too much work and I don't wanna write my own parser just to extract that info.

Reply:
Instead of metadata in each file, why not make a file index.json like this:
```
[
    { "title": "Promoting Lemmy", "date": "2021-08-09", "filename": "promoting-lemmy.md" },
    ...
]
```

Then we wouldnt have to mess with the source code for each new post, and could use more descriptive filenames. At the same time, we could use filenames to construct nicer urls like `/news/promoting-lemmy`, because `/news_item/3` is really ugly.

Reply:
I think there's a way for me to do this using filenames alone, I'll think of something.